src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { injectIntl, IntlShape } from 'react-intl';
+import BoxSign28 from '../../icon/logo/BoxSign28';
+import PlainButton, { PlainButtonProps } from '../../components/plain-button';
+import Tooltip, { TooltipPosition } from '../../components/tooltip';
+// @ts-ignore Module is written in Flow
+import messages from './messages';
+import './SidebarNavSign.scss';
+
+export type Props = PlainButtonProps & {
+    intl: IntlShape;
+    status?: string;
+};
+
+export function SidebarNavSign({ intl, status, ...rest }: Props) {
+    const label = status === 'active' ? messages.boxSignSignature : messages.boxSignRequest;
```

Comment:
Maybe pull out the text into a variable? Can have `messageObject` and `const label = intl.formatMessage(messageObject)`